roles/kubespray-defaults/defaults/main/checksums.yml

```diff
@@ -1169,3 +1169,15 @@ yq_checksums:
     4.40.5: sha256:a1df9d2b872fbb30583526bf4f37f737dc1913b28606dfc1dafeaf56a8862b3d
     4.40.4: sha256:c67379085a44558825a60a8af3b59b400852b168356070829bc0f45c70553f45
     4.40.3: sha256:2fe818a0b141913a41548e0e727267479d0f755221c73f9e304788c8e9139a45
+gateway_api_crds_checksums:
+  no_arch:
+    standard:
+      1.2.1: sha256:97598bf6ab3b33b9b5c5432bdd24de091e4e9c3aa0575ebb0710a2a19cd64d64
+      1.2.0: sha256:38ed055bb25dc580c0366899c0bed9b9e92dfcd1c180a569133f3946026cf102
+      1.1.0: sha256:c411805475d430a34242623a8e17153a7c40e946497bfd494e558b0d1a8858b3
+      1.0.0: sha256:23e4e1095c72a0587474f7fb3f85c319cdec77a083ab91237ffbdec1f1834d2a
+    experimental:
+      1.2.1: sha256:d3aa6723a3306770cffb601ee22af3d35da43acfa1ca547fc0d3bce08dad66e7
+      1.2.0: sha256:4369188e63b9ab5a35b5a83032c94d871159dece086b908b6ea18ea321ca06a9
+      1.1.0: sha256:10f322744a005d4e73e2b067e95fecd4cfec619dc7564930b488c296bfa3bec1
+      1.0.0: sha256:6c601dced7872a940d76fa667ae126ba718cb4c6db970d0bab49128ecc1192a3
```

Comment:
FYI, I don't think that's compatible with the way component_hash_update works. (checksums: arch: versions).

Would it be a hassle to instead have `gateway_api_standard_crds_checksums` and `gateway_api_experimental_crds_checksums` ?

Reply:
I didn't know there were other uses for lookup, thanks!